components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -156,13 +159,38 @@ export default function () {
                     </div>
                 </div>
             </div>
-            {BillingMode.canSetWorkspaceClass(teamBillingMode) && (
-                <SelectWorkspaceClass
-                    workspaceClass={project.settings?.workspaceClasses?.regular}
-                    enabled={BillingMode.canSetWorkspaceClass(teamBillingMode)}
-                    setWorkspaceClass={setWorkspaceClass}
-                />
-            )}
+            <div>
+                <h3 className="mt-12">Workspaces</h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">
+                    Choose the workspace machine type for your workspaces.
+                </p>
```

Comment:
**thought**: I find the section heading here unnecessary as if we're trying to fool users by advertizing a feature that does not exist, or they can not reach, especially since there are no other workspace settings, and we're not allowing users to close this alert. I'd drop the section heading, but I'll leave you and @jldec decide because of the relevant comment in https://github.com/gitpod-io/gitpod/issues/14831#issuecomment-1339392705. 🏓

Reply:
Thanks @gtsiolis. I agree that we wouldn't want to fool our users about anything.

Ping @jldec for final decision about the heading 🙂 🙏

Reply:
I think it's clearer with the heading Let's leave it in.

Reply:
DEAL—Let's go with that and change later if needed. 🤝